src/Raven.Server/Config/Categories/CoreConfiguration.cs

```diff
@@ -41,6 +43,11 @@ public class CoreConfiguration : ConfigurationCategory
         [ConfigurationEntry("DataDir", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public PathSetting DataDirectory { get; set; }
 
+        [Description("Determines what kind of security was chosen during setup.")]
+        [DefaultValue(SetupMode.Initial)]
```

Comment:
This should be `None` or some other neutral value. If the user removed the value from the settings, we don't want to reset things.